_includes/footer.html

```diff
@@ -1,18 +1,26 @@
 <footer class="footer">
   <div class="container container-footer">
-    <div class='row'>
+    <div class="row">
       <div class="col-xs-12 col-lg-4">
-        <a class='footer-logo' href="/">
-          <img src="/assets/images/logo/logo@2x.png" />
+        <a class="footer-logo" href="/">
+          <img src="/assets/logo/moveit_logo-white.png" />
         </a>
-        <p>Brought to you by <a href="https://picknik.ai/" target="_blank">PickNik Consulting</a>
-          <br /> and these <a href="http://moveit.ros.org/about/">awesome maintainers</a><br />
-          <a href="https://picknik.ai/donate-to-moveit" target="_blank">Donate today!</a>
+        <p>
+          Brought to you by
+          <a href="https://picknik.ai/" target="_blank">PickNik Consulting</a>
+          <br />
+          and these
+          <a href="http://moveit.ros.org/about/">awesome maintainers</a><br />
+          <a href="https://picknik.ai/donate-to-moveit" target="_blank"
+            >Donate today!</a
```

Comment:
Is this some new auto-format? I think it's weird that the closing tags are split in half.

Reply:
I fixed it